serverless/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "datadog-serverless-trace-mini-agent"
```

Comment:
Not sure I see the necessity of having this as a separate crate, instead of a bin inside the `trace-mini-agent` crate.

You can condition the import of tokio to a feature, that would be required in the bin definition

https://doc.rust-lang.org/cargo/reference/cargo-targets.html#binaries

Reply:
I think I'll leave it separate for now, for the sake of clarity especially as we add more serverless-specific code.